src/app/components/form/input/state/FormInputStateError.vue

```diff
@@ -9,9 +9,8 @@
         validationProperty in formInput &&
         formInput[validationProperty].$invalid)
     "
-    class="text-(--semantic-critic-text)"
+    class="py-1 font-semibold text-(--semantic-critic-text)"
```

Comment:
The text size in `FormInputState` should be 11px / use a `TypographyLabel` component that sets this and `font-semibold`.
Is `py-1` specified in the design somewhere?